packages/neon/neon/lib/src/widgets/cached_image.dart

```diff
@@ -2,18 +2,81 @@ part of '../../neon.dart';
 
 final _cacheManager = DefaultCacheManager();
 
-abstract class NeonCachedImage extends StatefulWidget {
-  const NeonCachedImage({
+typedef APIImageDownloader = Future<Uint8List> Function();
```

Comment:
The name looks a bit odd because of the double `I`'s but I don't have a better name

Reply:
just removed the API :)